pages/company/components/Story.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable react/no-unescaped-entities */
+import { Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { FC } from "react";
+import styles from "../company.module.scss";
+import { pollution } from "assets/png";
+import Image from "next/image";
+
+const story = [
+	{ title: "	✓ Founded in 2014" },
+	{ title: "	✓ Community of 125K+" },
+	{ title: "	✓ $33M in funding" },
+	{ title: "	✓ 500+ global meetups" },
+];
+export const Story: FC = () => {
+	return (
+		<div className={styles.story}>
+			<div className={styles.title}>
+				<Typography
+					variant="h4"
+					sx={{
+						sm: { textAlign: "center" },
+						md: { textAlign: "left" },
+					}}
+				>
+					<Typography fontSize="1.8rem" fontWeight={500} mb={3}>
+						OUR STORY
+					</Typography>
+					We’re going to become <br /> partners for long run.
+				</Typography>
+			</div>
+			<Box>
+				<Typography maxWidth={559} textAlign="center">
+					Lorem ipsum dolor sit amet consectetur, adipisicing elit. Et odit
+					quibusdam voluptates dolores pariatur porro ipsum nisi illo,
+					cupiditate iste, asperiores totam exercitationem?{" "}
+				</Typography>{" "}
+				<br />
+				<Box
+					className={styles.cards}
+					gridTemplateAreas={{ xs: '"a" "b" "c" "d"', sm: '"a b" "c d"' }}
+				>
+					{story.map((feat, i) => {
+						const textAlign = i % 2 === 0 ? "left" : "right";
```

Comment:
where is this `textAlign` getting used?

Reply:
in the mobile view the texts were aligning themselves alternatively left and right.

Reply:
yes, but `textAlign` is not getting used anywhere. I think its redundant cuz its not getting used

Reply:
ohk i will remove this

Reply:
resolved